src/DI/SentryExtension.php

```diff
@@ -23,7 +23,7 @@ class SentryExtension extends CompilerExtension
         self::PARAM_ENVIRONMENT => 'local',
         self::PARAM_USER_FIELDS => [],
         self::PARAM_PRIORITIES_MAPPING => [],
-        self::PARAM_TRACES_SAMPLE_RATE => 0,
+        self::PARAM_TRACES_SAMPLE_RATE => 0.0,
```

Comment:
Please test, what sample rate will be used when no `traces_sample_rate` is set in the config - whether it's `0.0` or whether it's `null`. If it's `0.0`, we'll need to change the default value to `null` also here.